config.yml

```diff
@@ -3,28 +3,68 @@ description: >-
   Welcome to GitHub! We're so glad you're here. We know it can look overwhelming
   at first, so we've put together a few of our favorite courses for people
   logging in for the first time
+preferences:
+- type: dropdown
+  name: language
+  label: Language
+  description: Choose your preferred language.
+  options:
+    - label: English
+      value: en 
+    - label: Español
+      value: es
 steps:
   - link: 'https://youtu.be/w3jLJU7DT5E'
     title: What is GitHub?
     description: >-
       Hey, it's your first time here, so let's not take any chances. Check out
       this short video to understand why people use GitHub.
+    translations:
+      es:
+        title: ¿Qué es GitHub?
+        description: >-
+         Hey, es tu primera vez aquí, así que no presupongamos nada. Mira este
+         breve video para entender por qué la gente usa GitHub.
   - course: introduction-to-github
     description: >-
       That video is pretty cool, right? If you want to learn how to use the
       workflow described in that video, take this course.
+    translations:
+      es:
+        description: >-
+        El video está bien, ¿verdad? Si quieres aprender a usar el flujo de 
+        trabajo descrito en el video, sigue este curso.
   - link: 'https://guides.github.com/introduction/git-handbook/'
     title: Git Handbook
     description: >-
       By now you are probably wondering what a Git is and why it is important in
       writing code. We have you covered with this short article on version
       control with Git.
+    translations:
+      es:
+        title: Manual de Git
+        description: >-
+        A estas alturas probablemente te estás preguntando qué es Git y por qué es
+        importante para escribir código. Aquí tienes un breve artículo sobre 
+        control de versiones con Git.
   - course: communicating-using-markdown
     description: >-
       GitHub is all about collaboration and we collaborate in issues in pull
       requests. Learn how to get your point across with the worlds easiest text
       formatting syntax.
+    translations:
+      es:
+        description: >-
+        GitHub va sobre todo de colaboración, y colaboramos en _issues_ que forman 
```

Comment:
Although we have been translating issues and PRs I am more into the not translating those concepts to be honest 😄  so to me this is a 👍